app/models/base_facility.rb

```diff
@@ -11,12 +11,6 @@ class BaseFacility < ApplicationRecord
 
   YES = 'YES'
 
-  APPROVED_SERVICES = %w[
-    MentalHealthCare
-    PrimaryCare
-    DentalServices
-  ].freeze
-
   HOURS_STANDARD_MAP = DateTime::DAYNAMES.each_with_object({}) { |d, h| h[d] = d }
```

Comment:
Since this is a constant and unlikely to change, I'd expect to see it written out so its clear what the constant is, instead of having to mentally (or literally) run the code. This is *not* data that's going to change so generating it seems like extra complexity.

Reply:
HOURS_STANDARD_MAP evaluates to `{"Sunday"=>"Sunday", "Monday"=>"Monday", "Tuesday"=>"Tuesday", "Wednesday"=>"Wednesday", "Thursday"=>"Thursday", "Friday"=>"Friday", "Saturday"=>"Saturday"}`, which seems off to me.

Reply:
It's odd for sure—but accurate. That same map is used across facility types, and this is the most common format. So certain types override it locally, but we can let this base transformation run on the other types without needing to add a bunch of conditionals to our parser. 

Also, changing the code that evaluates the map into a frozen hash!

Reply:
@annaswims if you search the code, it gets relooped over and downcased in one of the three places that uses it https://github.com/department-of-veterans-affairs/vets-api/blob/eeb0b2b9ee14ab377b251de45033693a2f1e7483/lib/facilities/vc_facility.rb#L35